typegate/tests/runtimes/python_wasi/python_wasi.ts

```diff
@@ -1,16 +1,16 @@
 // Copyright Metatype OÜ, licensed under the Elastic License 2.0.
 // SPDX-License-Identifier: Elastic-2.0
 
-import { Policy, t, typegraph } from "@typegraph/sdk/mod.ts";
-import { PythonRuntime } from "@typegraph/sdk/runtimes/python.ts";
+import { Policy, t, typegraph } from "@typegraph/sdk/mod.js";
+import { PythonRuntime } from "@typegraph/sdk/runtimes/python.js";
 import outdent from "outdent";
 
 const tpe = t.struct({
   "a": t.string(),
   "b": t.list(t.either([t.integer(), t.string()])),
 });
 
-typegraph("python_wasi", (g) => {
+typegraph("python_wasi", (g: any) => {
```

Comment:
can we type this instead?

Reply:
I have issues on deno, it has problems typing with `*.d.ts` files when functions as arguments are involved. But on node (even pure js) the type hints works perfectly.